src/templates/static/get_started/binary_options_mt5.jsx

```diff
@@ -115,6 +113,42 @@ const BinaryOptionsForMT5 = () => (
 
                     <SeparatorLine invisible show_mobile className='gr-padding-10' />
                 </Section>
+                <Section id='type-of-trades' header={it.L('Type of trades')}>
+                    <HeaderSecondary header={it.L('Up/Down')} />
+                    <div className='gr-row'>
+                        <img className='responsive gr-6 gr-12-m center' src={it.url_for('images/pages/get-started/mt5/type-of-trades/up.svg')} />
```

Comment:
i'm not sure about this, bcause i cant see any images in this page to have `alt` tags, and have to ask copywriter for what kind of text should I write there `or` i can just add text for example `How to trade in binary MT5 step 1` @negarn

Reply:
it's fine, we don't follow this practice for now, although it's a good idea.

Reply:
I still think it would be an easy thing to add here for SEO and accessibility? Didn't we talk about starting to add `alt` tags "from now on" previously 😄